pinax/stripe/webhooks.py

```diff
@@ -96,7 +96,8 @@ def validate(self):
                 cls=stripe.StripeObjectEncoder
             )
         )
-        self.event.valid = self.event.webhook_message["data"] == self.event.validated_message["data"]
+        # Notice "data" may contain a "previous_attributes" section
+        self.event.valid = "object" in self.event.webhook_message["data"] and "object" in self.event.validated_message["data"] and self.event.webhook_message["data"]["object"] == self.event.validated_message["data"]["object"]
```

Comment:
Looks wrong: it should also be in the validate (i.e. fetched) event then..?!

Reply:
Yes, odd isn't it; but i actually found this when making a change on the subscription using the Stripe dashboard. The initial event contains info about what has changed while the fetched event does not...

Reply:
That would be worth a separate test then, using the actual data/fixtures.

Reply:
Regarding your last comment on the PR: please extract it into a separate PR.